test/securityHelper_test.go

```diff
@@ -0,0 +1,74 @@
+// define/securityHelpers
```

Comment:
This file has not been `gofmt` , I can't review it as-is, it's unreadable.

Reply:
that's fixed